store/views.py

```diff
@@ -62,21 +69,68 @@ def loanBookView(request):
     If yes, then set the message to 'success', otherwise 'failure'
     '''
     # START YOUR CODE HERE
-    book_id = None # get the book id from post data
+    book_id = request.POST['bid']  # get the book id from post data
+    booksOfThatId = BookCopy.objects.filter(book=book_id, status=True)
+    if booksOfThatId.count() != 0:
+        singleBook = booksOfThatId[0]
+        singleBook.borrower = request.user
+        singleBook.borrow_date = date.today()
+        singleBook.status = False
+        singleBook.save()
+        response_data['message'] = "success"
 
+    else:
+        response_data['message'] = "faliure"
 
     return JsonResponse(response_data)
 
+
 '''
 FILL IN THE BELOW VIEW BY YOURSELF.
 This view will return the issued book.
 You need to accept the book id as argument from a post request.
 You additionally need to complete the returnBook function in the loaned_books.html file
 to make this feature complete
-''' 
+'''
 @csrf_exempt
 @login_required
 def returnBookView(request):
-    pass
+    response_data = {
+        'message': None,
+    }
+    key = request.POST['bId']
+    try:
+        singleBook = BookCopy.objects.get(pk=key)
+        singleBook.borrower = None
+        singleBook.borrow_date = None
+        singleBook.status = True
+        singleBook.save()
+        response_data['message'] = "success"
+    except:
+        response_data['message'] = "faliure"
+
+    return JsonResponse(response_data)
```

Comment:
There must be a validation in the backend when a user is returning the book, to make sure that he has only borrowed the book. Otherwise, a simple POST request will make the BookCopy to be returned, and would set its status as True.